src/types/aws/ap/index.ts

```diff
@@ -89,13 +89,22 @@ export type EndowmentOption = Pick<Endowment, "id" | "name">;
 
 //most are optional except id, but typed as required to force setting of default values - "", [], etc ..
 export type EndowmentProfileUpdate = Except<
-  Ensure<Endowment, "hide_bg_tip" | "slug" | "card_img">,
-  "endow_designation" | "fiscal_sponsored"
+  Required<Endowment>,
+  "endow_designation" | "fiscal_sponsored" | "receiptMsg" | "program"
 > & {
   endow_designation: EndowDesignation | "";
-  program_id: string;
 };
 
+export type EndowmentSettingsUpdate = Pick<
```

Comment:
future: maybe we could also add slug and tip opt out in this settings update

Reply:
> future: maybe we could also add slug and tip opt out in this settings update

@ap-justin That would be a good improvement over edit profile. Since they are not directly related to the public profile pages. :100: